elegant/gui/pose_annotation.py

```diff
@@ -117,6 +125,9 @@ def init_widget(self):
         lock_warp.toggled.connect(self.set_locked)
         self._add_row(layout, lock_warp, self.fine_mode, self.reverse_button)
 
+        
```

Comment:
Generally do a `git diff` before checkin to make sure that stray changes unrelated to the code (such as these random extra spaces) don't accidentally get carried along.